test/operators.js

```diff
@@ -199,36 +199,7 @@ test('execute and(hasRoot(msgkey))', (t) => {
         t.error(err, 'no err')
 
         db.query(
```

Comment:
finally found something to nitpick, the test needs to be called `execute hasRoot(msgkey)` ;-)

Reply:
Pushed up a change for this to try and run the tests again :)